runtime/lua/vim/filetype/detect.lua

```diff
@@ -1019,18 +1019,20 @@ end
 -- This function checks for an assembly comment in the first ten lines.
 -- If not found, assume Progress.
```

Comment:
```suggestion
-- This function checks for an assembly comment or a SWIG keyword or verbatim block in the first 50 lines.
-- If not found, assume Progress.
```

Maybe it'd be good to include the comments added above the new `ft_swig` vars too